### PR TITLE
Reformat the code base with Eclipse

### PR DESCRIPTION
**Pull request by bsataric**
This PR has been created in order to change file formatting of certain files according to eclipse IDE formatting. This will help in easy comparison with task addition branch which has been formatted in the same way by Eclipse.

No actual changes to the source codes have been made appart to the formatting (indendation, bracket positioning etc).

### CODE REVIEW

File: src/confdb/converter/ConverterFactory.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	private String classHeader = "Ascii";
 	private boolean untrackedFlag = false;
 	private String additionalParameter4ConfigurationWriter = null;
-
+	//DUMY COMMENT
```
Comment (fwyzard) [inline src/confdb/converter/ConverterFactory.java]: I'm curious... why this ?

File: bin/.gitignore [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ext/signed/
+ext/cachedir/
+lib/
+.*.swp
+*.class
+*.pyc
+/browser/
+/build.xml
+/createKeyStore
+/deploy.sh
```
Comment (fwyzard) [inline bin/.gitignore]: these files are part of the repository, why do you propose to ignore them ?

File: src/confdb/converter/ConverterFactory.java (hunk 1 above)
Comment (bsataric) [reply src/confdb/converter/ConverterFactory.java]: I added it so I could check some problems that we had trying to compare branches before this.  Just to see weather commit reflects on the branch in comparison. It should be removed.

File: bin/ChangeLog [hunk 3]
```diff
@@ -0,0 +1,2612 @@
+2017-06-08 Andrea Bocci <andrea.bocci@cern.ch>
```
Comment (fwyzard) [inline bin/ChangeLog]: why do you copy this from the root directory ?

File: bin/README [hunk 4]
```diff
@@ -0,0 +1,94 @@
+----------------
```
Comment (fwyzard) [inline bin/README]: why do you copy this from the root directory ?

File: src/confdb/data/ConfigInfo.java [hunk 5]
```diff
@@ -5,238 +5,238 @@
 import java.util.ArrayList;
 import java.util.Collections;
 
-
 /**
- * ConfigInfo
- * ----------
+ * ConfigInfo ----------
```
Comment (fwyzard) [inline src/confdb/data/ConfigInfo.java]: mhm, no, this was supposed to stay on the next line

File: bin/.gitignore (hunk 2 above)
Comment (bsataric) [reply bin/.gitignore]: I haven't changed gitignore by hand from the origiinal code I branched off. I don't know why it became chaged meanwhile. I wil change it back to original as in master. My guess is that it has to do something with project being repeatedly opened in Eclipse.

File: src/confdb/data/Configuration.java [hunk 6]
```diff
@@ -7,1573 +7,1588 @@
 import java.util.StringTokenizer;
 
 /**
- * Configuration
- * -------------
+ * Configuration -------------
```
Comment (fwyzard) [inline src/confdb/data/Configuration.java]: same here

File: src/confdb/data/ConfigurationModifier.java [hunk 7]
```diff
@@ -3,976 +3,917 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 
-
 /**
- * ConfigurationModifier
- * ---------------------
+ * ConfigurationModifier ---------------------
```
Comment (fwyzard) [inline src/confdb/data/ConfigurationModifier.java]: same here

File: src/confdb/data/ConfigurationModifier.java [hunk 8]
```diff
@@ -3,976 +3,917 @@
...
+ * 
  * @author Philipp Schieferdecker
  *
- * retrieve only selected parts of a configuraiton,
- * after applying configurable filter decisions.
+ *         retrieve only selected parts of a configuraiton, after applying
```
Comment (fwyzard) [inline src/confdb/data/ConfigurationModifier.java]: this in indentation does not seem correct

File: bin/ChangeLog (hunk 3 above)
Comment (bsataric) [reply bin/ChangeLog]: Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

File: bin/README (hunk 4 above)
Comment (bsataric) [reply bin/README]: Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

File: src/confdb/data/ConfigInfo.java (hunk 5 above)
Comment (bsataric) [reply src/confdb/data/ConfigInfo.java]: OK. Will revert manually.

File: src/confdb/data/Configuration.java (hunk 6 above)
Comment (bsataric) [reply src/confdb/data/Configuration.java]: OK. Will revert manually.

File: src/confdb/data/ConfigurationModifier.java (hunk 7 above)
Comment (bsataric) [reply src/confdb/data/ConfigurationModifier.java]: OK. Will revert manually.

File: src/confdb/data/DatabaseEntry.java [hunk 9]
```diff
@@ -3,49 +3,56 @@
 import java.io.Serializable;
 
 /**
- * DatabaseEntry
- * --------------
+ * DatabaseEntry --------------
```
Comment (fwyzard) [inline src/confdb/data/DatabaseEntry.java]: same here

File: src/confdb/data/DatabaseEntry.java [hunk 10]
```diff
@@ -3,49 +3,56 @@
...
+ * 
  * @author Philipp Schieferdecker
  *
- * Class to keep track of the status of a component in the
- * database: cache the primary key for objects which are already in
- * the DB and have not changed, indicate those object which are either
- * not yet in the DB or have changed (and therefore need to have a new
- * copy saved.
+ *         Class to keep track of the status of a component in the database:
```
Comment (fwyzard) [inline src/confdb/data/DatabaseEntry.java]: indentation ?

File: src/confdb/gui/CreateStreamDialog.java [hunk 11]
```diff
@@ -11,197 +11,195 @@
 import confdb.data.Configuration;
 
 /**
- * CreateStreamDialog
- * ------------------
+ * CreateStreamDialog ------------------
```
Comment (fwyzard) [inline src/confdb/gui/CreateStreamDialog.java]: same here

File: src/confdb/data/ConfigurationModifier.java (hunk 8 above)
Comment (bsataric) [reply src/confdb/data/ConfigurationModifier.java]: Will correct. I guess Ecliipse is not perfect.

**Comment by fwyzard** [comment]
I stopped checking after the first few files... could you check the formatting of the comments ?
Some of them do not look right.

The changes in the actual code look fine.

File: src/confdb/data/DatabaseEntry.java (hunk 10 above)
Comment (bsataric) [reply src/confdb/data/DatabaseEntry.java]: Will correct. I guess Ecliipse is not perfect.

File: src/confdb/data/DatabaseEntry.java (hunk 9 above)
Comment (bsataric) [reply src/confdb/data/DatabaseEntry.java]: OK. Will revert manually.

File: src/confdb/gui/CreateStreamDialog.java (hunk 11 above)
Comment (bsataric) [reply src/confdb/gui/CreateStreamDialog.java]: OK. Will revert manually.

**Comment by fwyzard** [comment]
There is also a new binary file, ` bin/antlib/ant-contrib-1.0b3.jar`.
If it was not intended, could you delete it ?

**Comment by bsataric** [comment]
> I stopped checking after the first few files... could you check the formatting of the comments ?
> Some of them do not look right.
> 
> The changes in the actual code look fine.

When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?

**Comment by fwyzard** [comment]
> When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?

For example
  - https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-a19dc91caec343a829f3aecbefd43641L8-R8
  - https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-ebe8f24fd70b33027bdbec97406b5176L17-R17

The bulk of the code and the comments throughout the body look fine, from a quick glance.

**Comment by bsataric** [comment]
> > When you say comments - you mean comments in the header of the changed files or all coments anywhere in the files?
> 
> For example
> 
> * https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-a19dc91caec343a829f3aecbefd43641L8-R8
> * https://github.com/cms-sw/hlt-confdb/pull/13/files?diff=unified&w=1#diff-ebe8f24fd70b33027bdbec97406b5176L17-R17
> 
> The bulk of the code and the comments throughout the body look fine, from a quick glance.

OK. I pushed changes you proposed. Can you check if this is what you wanted? Especially regarding gitignore files because a lot of files have been added. Thanks!

File: bin/.gitignore [hunk 12]
```diff
@@ -0,0 +1,18 @@
+classes/
```
Comment (fwyzard) [inline bin/.gitignore]: My original comment was probably wrong, sorry about that.

Why do you add this file at all ?
It should already be there in the top directory.

File: bin/ChangeLog (hunk 3 above)
Comment (fwyzard) [reply bin/ChangeLog]: Still not deleted ?

File: bin/README (hunk 4 above)
Comment (fwyzard) [reply bin/README]: Still not deleted ?

**Comment by fwyzard** [comment]
`bin/antlib/ant-contrib-1.0b3.jar` and other new files are still present.
Can you remove those that are clearly there by mistake ?
About the additional xml files, which ones are actually needed/used ?

**Comment by fwyzard** [comment]
Actually, it looks like there is a whole new `bin/browser` directory ?

Can you please remove all the new files (unless they are needed by Eclipse to build the project),  and squash away all intermediate commits to remove them from the git history ?

**Comment by bsataric** [comment]
> Actually, it looks like there is a whole new `bin/browser` directory ?
> 
> Can you please remove all the new files (unless they are needed by Eclipse to build the project), and squash away all intermediate commits to remove them from the git history ?

OK I hope I deleted all uncessary files/folders (I tried to mimic what is already on ConfDB repo and eliminate all that is not already there). I also squashed commits into 1.

File: src/confdb/data/ConfigInfo.java [hunk 13]
```diff
@@ -5,238 +5,238 @@
 import java.util.ArrayList;
 import java.util.Collections;
 
-
 /**
- * ConfigInfo
+ * ConfigInfo
```
Comment (fwyzard) [inline src/confdb/data/ConfigInfo.java]: sorry for being pedantic - could you avoid adding the extra space at the end of the line ?
(here and in the other files)

**Comment by fwyzard** [comment]
The rest looks good, thanks for the clean up !

**Pull request merged**